### PR TITLE
Change description and add source to metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**
Metadata changes to adapt to discourse as per https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366

### CODE REVIEW

**Review by gtrkiller — DISMISSED** [review]
LGTM

**Comment by amandahla** [comment]
Could you add *.rock to .gitignore?

**Comment by arturo-seijas** [comment]
> *.rock



> Could you add *.rock to .gitignore?

Done. Thanks

**Review by weiiwang01 — DISMISSED** [review]

**Review by nrobinaubertin — APPROVED** [review]

**Review by weiiwang01 — DISMISSED** [review]

**Review by gtrkiller — APPROVED** [review]
LGTM

**Review by amandahla — APPROVED** [review]
LGTM

**Review by weiiwang01 — APPROVED** [review]

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by weiiwang01 — DISMISSED** [review]

**Review by gtrkiller — APPROVED** [review]
LGTM

File: .trivyignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CVE-2022-40897
```
Comment (jdkandersson) [inline .trivyignore:1]: Can we fix these?

File: pyproject.toml [hunk 2]
```diff
@@ -44,3 +44,8 @@ docstring-convention = "google"
 copyright-check = "True"
 copyright-author = "Canonical Ltd."
 copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
+
+[tool.mypy]
```
Comment (jdkandersson) [inline pyproject.toml:48]: We have some additional settings defined in this standard: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints, should they be added too?

**Review by merkata — DISMISSED** [review]

File: pyproject.toml (hunk 2 above)
Comment (arturo-seijas) [reply pyproject.toml:48]: There's an item in the backlog to address linting properly. IMHO this is out of scope

File: .trivyignore (hunk 1 above)
Comment (arturo-seijas) [reply .trivyignore:1]: There's an item in the backlog to address linting properly. IMHO this is out of scope

File: .github/workflows/integration_test.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@debug
```
Comment (amandahla) [inline .github/workflows/integration_test.yaml:8]: Debug?

Comment (arturo-seijas) [reply .github/workflows/integration_test.yaml]: Ups. Thx

**Review by amandahla — APPROVED** [review]
LGTM

**Review by yanksyoon — APPROVED** [review]
LGTM!

File: tox.ini [hunk 4]
```diff
@@ -1,72 +1,110 @@
...
 skipsdist=True
-envlist = lint, unit, static, coverage-report
 skip_missing_interpreters = True
+envlist = lint, unit, static, coverage-report
 
 [vars]
 src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
+plugins_path = {toxinidir}/indico_rock/plugins/
+;lib_path = {toxinidir}/lib/charms/operator_name_with_underscores
```
[the remark this answers is not in the document]
Comment (arturo-seijas) [reply tox.ini:10]: It's commented out ;) That's the content of the tox template for charmcraft

**Pull request merged**